KC7-Foundation/kc7_scoreboard
# "Cleaned up" models.py and views.py

### Description
This was not a refactor of the code. This was a simple cleanup going through to add docstrings, get rid of unused imports, and restructure these files for later use. This is laying the base for us to do some heavy refactoring, like moving models into individual folders and separating views logically (primarily for readability and testing).

### Testing
1. Make sure simple docstring explanations are correct
2. Run the project to ensure that I did not accidentally break any core functionality
